site/content/integrate/admin-guide/_index.md

```diff
@@ -0,0 +1,167 @@
+---
+title: "Mattermost Integrations and Developer Toolkit"
+heading: "Integrations and Toolkit Overview"
+description: "The Mattermost Developer Toolkit gives community and in-house customer developers the power to integrate deeply with, and extend, Mattermost servers and clients in order to meet their particular requirements - without “reinventing the wheel”."
+weight: 90
+---
+
+## Better Control for System Admins
+
+The main benefits of the Mattermost Developer Toolkit from a system administration perspective are first, that it puts all of the resources and information that developers will need in one central location and second, it simplifies integrations with Mattermost.
+
+Consequently this will save time for system administrators as well as developers.
+
+## Features of the Developer Toolkit
+
+Below is a list of planned features of the developer toolkit with estimated time of delivery.
+
+1. Completed:
+
+ - Webhooks and slash commands to allow easy, low-effort extension and integration.
+ - Mattermost HTTP REST APIv4 allowing for much more powerful server interaction.
+ - Mattermost webapp moved over to Redux infrustructure.
+ - API developer token to provide a simple method to authenticate to the Mattermost REST API.
+ - The ability to build webapp client plugins to override existing UI components (replace posts with your custom components, use your own video services etc.), modify/extend client drivers to interact with custom server API endpoints, and add whole new UI views in predetermined places.
+ - The ability to build server plugins to hook directly into server events (e.g., new post events, user update events, etc.), have some form of database access (possibly access to certain tables, and the ability to create new tables) and to add custom endpoints to extend the Mattermost REST API.
+
+2. Upcoming:
+
+ - The ability to build plugins similar to the webapp but for React Native apps for iOS and Android.
+ - A system or architecture to combine the above plugins into one easy-to-share and easy-to-install package.
+ - A market or directory to find official and/or certified by Mattermost plugins and a process to get your plugin certified.
+ - A guide or system to enable the embedding of Mattermost into other apps as a chat service.
```

Comment:
I'm hesitant to publish this as a planed item given that it's not on the roadmap